docs/source/tutorial.rst

```diff
@@ -2,24 +2,12 @@
 Tutorial: Intro to BentoML
 ==========================
 
-BentoML is a python-first, efficient and flexible framework for machine learning model
-serving. It lets data scientists save and version trained models in a standardized
-format and unifies how a saved model can be accessed for serving. This enables ML
-engineers to easily use the saved models for building online prediction services or
-batch inference jobs.
-
-BentoML also helps with defining the APIs, environments, and dependencies for running a
-model, providing a build tool that encapsulates all model artifacts, source code, and
-dependencies into a self-contained format :code:`Bento`, which is designed to be DevOps
-friendly and ready for production deployment - just like docker but designed for ML
-models.
-
 What are we building?
```

Comment:
Can we incorporate the time commitment expected from users for the tutorial? I find it encouraging when documentation mention that the tutorial is expected to take 5, 10, or 15 minutes.

Reply:
We should keep the tutorial in under 10 minutes.